static/src/javascripts/projects/common/modules/experiments/tests/adblocking-response.js

```diff
@@ -54,20 +52,18 @@ define([
         };
 
         var isQualified = function () {
```

Comment:
This should probably be in `canRun` but you seem to have complex requirements, this should be discussed with @desbo before we put the actual test live